src/test/java/com/google/sps/data/AvailabilityServletTest.java

```diff
@@ -67,6 +78,8 @@ public void validAvailabilityServletRequest() throws IOException {
   @Test
   public void invalidAvailabilityServletRequest() throws IOException {
```

Comment:
How is this invalid? Please put in a comment.
Also, since we're going to be getting availability too from the Servlet, we should specify in the test name if it's a get, put, or post.

Reply:
We are not actually getting availability from the servlet. This is because of how the AvailabilityTimeSlotGenerator and its corresponding jsp page are set up.

Reply:
Added a comment